Hope/.claude/skills/hope-development/SKILL.md

```
@@ -7,82 +7,117 @@ description: Development guidelines for the Hope 3D experience project. Use when
+
+  useFrame((state, delta) => {
+    // Animation logic
+  })
+
+  return <mesh ref={meshRef}>{/* ... */}</mesh>
 }
 ```
 
-### GSAP Animations
+### GSAP Animations with Hooks
 ```typescript
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)